zgw_consumers/models/services.py

```diff
@@ -113,6 +126,22 @@ def clean(self):
                 {"header_key": _("If header_value is set, header_key must also be set")}
             )
 
+    @property
+    def get_health_check_indication(self) -> bool:
```

Comment:
Python code style thing - making a property that still has the `def get_` is very confusing, because if I see `obj.get_health_check_indication` in the code, that looks like someone forgot the parentheses.

If you're going to be using properties, make the name reflect that:

```suggestion
    @property
    def health_check_indication(self) -> bool:
```